### ISSUE TITLE
Country field in checkout page

### ISSUE THREAD

**Issue by adem-demir** [woocommerce/woocommerce#21518]
**Describe the bug**
When we add an additional field in the checkout page, we get the billing_country_field on the top of the page above the name and surname of the customer. Also when you don't use any additional field the billing_country_field is showing for few seconds in the top and goes down. 

**To Reproduce**
1. Go to 'WooCommerce-MyParcel' in plugins and install/activate the plugin
2. See error on top of the page

**Screenshots**
![testomgeving adem afrekenen 2018-10-10 13-35-06](https://user-images.githubusercontent.com/37372250/46733744-729cc100-cc91-11e8-9442-7fc3f6d11461.jpg)

**Expected behavior**
The billing_country_field must be on the same postion as the additional plugin is active.

**Isolating the problem (mark completed items with an [x]):**
- [x ] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [x ] I can reproduce this bug consistently using the steps above.

**WordPress Environment**
<details>
```
`
### WordPress Environment ###

Home URL: http://localhost:8888/woocommerce
Site URL: http://localhost:8888/woocommerce
WC Version: 3.5.0
Log Directory Writable: ✔
WP Version: 4.9.8
WP Multisite: –
WP Memory Limit: ❌ 40 MB - Wij adviseren een minimum geheugen van 64MB. Bekijk: Verhogen van het geheugen toegekend aan PHP
WP Debug Mode: –
WP Cron: ✔
Language: nl_NL
External object cache: –

### Server Environment ###

Server Info: Apache/2.2.34 (Unix) mod_wsgi/3.5 Python/2.7.13 PHP/7.1.12 mod_ssl/2.2.34 OpenSSL/1.0.2j DAV/2 mod_fastcgi/2.4.6 mod_perl/2.0.9 Perl/v5.24.0
PHP Version: 7.1.12 - We raden je aan om PHP-versie 7.2 of hoger te gebruiken voor betere prestaties en beveiliging. Hoe je PHP-versie bijwerken
PHP Post Max Size: 32 MB
PHP Time Limit: 30
PHP Max Input Vars: 1000
cURL Version: 7.52.1
OpenSSL/1.0.2j

SUHOSIN Installed: –
MySQL Version: 5.6.38
Max Upload Size: 32 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 3.5.0
WC Database Prefix: wp_
MaxMind GeoIP Database: ✔
Totale databasegrootte: 5.37MB
Database datagrootte: 3.81MB
Database index grootte: 1.56MB
wp_woocommerce_sessions: Data: 0.02MB + Index: 0.03MB
wp_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB
wp_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB
wp_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.05MB
wp_woocommerce_order_items: Data: 0.02MB + Index: 0.02MB
wp_woocommerce_order_itemmeta: Data: 0.08MB + Index: 0.09MB
wp_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB
wp_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB
wp_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB
wp_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB
wp_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB
wp_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB
wp_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB
wp_woocommerce_log: Data: 0.02MB + Index: 0.02MB
wp_commentmeta: Data: 0.02MB + Index: 0.03MB
wp_comments: Data: 0.08MB + Index: 0.09MB
wp_layerslider: Data: 0.02MB + Index: 0.00MB
wp_layerslider_revisions: Data: 0.02MB + Index: 0.00MB
wp_links: Data: 0.02MB + Index: 0.02MB
wp_options: Data: 1.09MB + Index: 0.05MB
wp_postmeta: Data: 1.52MB + Index: 0.64MB
wp_posts: Data: 0.31MB + Index: 0.06MB
wp_revslider_css: Data: 0.13MB + Index: 0.00MB
wp_revslider_layer_animations: Data: 0.02MB + Index: 0.00MB
wp_revslider_navigations: Data: 0.02MB + Index: 0.00MB
wp_revslider_sliders: Data: 0.02MB + Index: 0.00MB
wp_revslider_slides: Data: 0.02MB + Index: 0.00MB
wp_revslider_static_slides: Data: 0.02MB + Index: 0.00MB
wp_termmeta: Data: 0.02MB + Index: 0.03MB
wp_terms: Data: 0.02MB + Index: 0.03MB
wp_term_relationships: Data: 0.02MB + Index: 0.02MB
wp_term_taxonomy: Data: 0.02MB + Index: 0.03MB
wp_usermeta: Data: 0.02MB + Index: 0.03MB
wp_users: Data: 0.02MB + Index: 0.05MB
wp_wc_download_log: Data: 0.02MB + Index: 0.03MB
wp_wc_webhooks: Data: 0.02MB + Index: 0.02MB

### Post Type Counts ###

attachment: 98
carousels: 8
nav_menu_item: 19
page: 19
portfolio_page: 24
post: 14
product: 40
revision: 4
shop_order: 106
slides: 2
testimonials: 6

### Security ###

Secure connection (HTTPS): ❌
					Jouw winkel gebruikt geen HTTPS. Leer meer over HTTPS en SSL-certificaten.
Hide errors from visitors: ✔

### Active Plugins (11) ###

LayerSlider WP: door Kreatura Media – 6.7.6
Envato Market: door Envato – 2.0.1
WPBakery Page Builder: door Michael M - WPBakery.com – 5.5.2
Qode Instagram Widget: door Qode Themes – 1.2.1
Qode Twitter Feed: door Qode Themes – 1.1
Slider Revolution: door ThemePunch – 5.4.8
WC Duplicate Order: door Jamie Gill – 1.5 – Niet getest met de actieve versie van WooCommerce
WooCommerce MyParcel: door Richard Perdaan – 3.0.2 – Niet getest met de actieve versie van WooCommerce
WooCommerce - Store Toolkit: door Visser Labs – 1.9.2 – Niet getest met de actieve versie van WooCommerce
WooCommerce: door Automattic – 3.5.0-rc.1
WP Rollback: door WordImpress – 1.5.1

### Settings ###

API Enabled: ✔
Force SSL: –
Currency: EUR (€)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)


### WC Pages ###

Winkelbasis: #4 - /winkel/
Winkelmand: #5 - /winkelmand/
Afrekenen: #6 - /afrekenen/
Mijn account: #7 - /mijn-account/
Algemene voorwaarden: ❌ Pagina niet ingesteld

### Theme ###

Name: Bridge
Version: 16.8
Author URL: http://www.qodethemes.com/
Child Theme: ❌ – Als je wijzigingen aanbrengt in WooCommerce of in een hoofdthema dat je niet zelf gebouwd hebt
raden we het gebruik van een child thema aan. Zie: Hoe maak je een child thema

WooCommerce Support: ✔

### Templates ###

Archive Template: Jouw thema heeft een woocommerce.php bestand. Je zal het woocommerce/archive-product.php template niet kunnen overschrijven omdat woocommerce.php prioriteit heeft over archive-product.php. Dit is bedoeld om weergave-problemen te voorkomen.
Overrides: bridge/woocommerce/cart/cart-empty.php versie 3.1.0 is verouderd. De hoofdversie is 3.5.0
bridge/woocommerce/cart/cart-shipping.php versie 3.2.0 is verouderd. De hoofdversie is 3.5.0
bridge/woocommerce/cart/cart-totals.php
bridge/woocommerce/cart/cart.php versie 3.4.0 is verouderd. De hoofdversie is 3.5.0
bridge/woocommerce/cart/shipping-calculator.php versie 3.4.0 is verouderd. De hoofdversie is 3.5.0
bridge/woocommerce/checkout/form-coupon.php versie 3.4.0 is verouderd. De hoofdversie is 3.4.4
bridge/woocommerce/checkout/thankyou.php
bridge/woocommerce/content-product.php
bridge/woocommerce/content-single-product.php
bridge/woocommerce/global/form-login.php
bridge/woocommerce/global/quantity-input.php
bridge/woocommerce/loop/add-to-cart.php
bridge/woocommerce/loop/loop-start.php
bridge/woocommerce/loop/pagination.php
bridge/woocommerce/loop/rating.php
bridge/woocommerce/loop/result-count.php
bridge/woocommerce/loop/sale-flash.php
bridge/woocommerce/myaccount/form-login.php versie 3.4.0 is verouderd. De hoofdversie is 3.5.0
bridge/woocommerce/myaccount/form-lost-password.php
bridge/woocommerce/myaccount/my-address.php
bridge/woocommerce/myaccount/my-orders.php
bridge/woocommerce/single-product/add-to-cart/external.php
bridge/woocommerce/single-product/add-to-cart/grouped.php
bridge/woocommerce/single-product/add-to-cart/simple.php
bridge/woocommerce/single-product/add-to-cart/variable.php
bridge/woocommerce/single-product/related.php
bridge/woocommerce/single-product/sale-flash.php
bridge/woocommerce/single-product/tabs/additional-information.php
bridge/woocommerce/single-product/tabs/description.php
bridge/woocommerce/single-product/tabs/tabs.php
bridge/woocommerce/single-product/up-sells.php

Outdated Templates: ❌
					
					
						Leer hoe te updaten

### Templates ###

thumbnail: 
medium: 
medium_large: 
large: 
portfolio-square: 
portfolio-portrait: 
portfolio-landscape: 
menu-featured-post: 
qode-carousel_slider: 
portfolio_slider: 
portfolio_masonry_regular: 
portfolio_masonry_wide: 
portfolio_masonry_tall: 
portfolio_masonry_large: 
portfolio_masonry_with_space: 
latest_post_boxes: 
woocommerce_thumbnail: 
woocommerce_single: 
woocommerce_gallery_thumbnail: 
shop_catalog: 
shop_single: 
shop_thumbnail: 
`
```
</details>

**Comment by claudiulodro** [comment]
Some behavior seems to have changed from 3.4 to 3.5, as the fields are in the correct place in 3.4 but not in 3.5. The behavior should be kept the same in 3.5, as it's a breaking change.

**Comment by claudiulodro** [comment]
I've investigated this. It appears to be an issue in the MyParcel plugin, as [the country is rendered at the top on purpose](https://github.com/myparcelnl/woocommerce/blob/ef63d1ee3097d6acd65b596ef8acb61443d6f45b/includes/class-wcmp-nlpostcode-fields.php#L265). 

This issue was not visible in the MyParcel plugin before because there was an automatic refresh/validation of the checkout fields when the Checkout page was first loaded that was removed in #20655. Hope this clears things up.
